Twig/Extension/CKEditorExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
```

Comment:
I'm sorry. Sonata header is missing. :-)

I'm suprised that it's not configured on .php_cs...